assembly/onpremises-ide-packaging-war-platform-api/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiServletModule.java

```diff
@@ -93,7 +93,9 @@ protected void configureServlets() {
                "/permissions",
                "/permissions/*",
                "/preferences",
-               "/preferences/*")
+               "/preferences/*",
+               "/system/ram",
```

Comment:
Seems that you don't need it. Mb it would be better to add this mapping when there will be some method that require it?